<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardPanel</class>
 <widget class="QMainWindow" name="CardPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CardPanel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startNewGameButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="passButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>610</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>496</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>553</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="com1Card1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="com1Card2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="com1Card3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="com2Card1">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>130</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="com2Card2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>240</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="com2Card3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>390</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerName">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>莊家</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="com1Name">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>電腦</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="com2Name">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>280</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>電腦</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="playerName">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>玩家</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_N"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="separator"/>
    <addaction name="action_L"/>
    <addaction name="separator"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>紀錄</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="separator"/>
    <addaction name="action_C"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="action_R"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_N">
   <property name="text">
    <string>新遊戲(N)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>儲存紀錄(S)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>讀取紀錄(L)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>離開遊戲(Q)</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>歷史紀錄(H)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>勝率圖表(C)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>規則(R)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
